src/App.js

```diff
@@ -1,8 +1,30 @@
 import React from "react";
 import "./index.css";
+import Contacts from "./Components/Contacts";
+import UserProfile from "./Components/UserProfile";
+import Posts from "./Components/Posts";
+import NavBar from "./Components/NavBar";
+import Post from "./Components/Post";
+import ContactUserCard from "./Components/ContactUserCard";
 
 const App = () => {
-  return <p>Hello, world!</p>;
+  return (
+    <div className="App">
+      <NavBar />
+      <UserProfile />
+      <Posts>
+        <Post />
+        <Post />
+        <Post />
+      </Posts>
+      <Contacts>
+        <ContactUserCard />
+        <ContactUserCard />
+        <ContactUserCard />
+        <ContactUserCard />
+      </Contacts>
+    </div>
+  );
```

Comment:
```suggestion
  return (
    <div className="App">
      <NavBar />
      <UserProfile />
      <Posts />
      <Contacts />
    </div>
  );
```
Since you're already placing four `<ContactUserCard />`  components inside of Contacts.js, you don't need to use them again here in App.js. The same goes for your three <Post /> components.